elasticsearch/elasticsearch.d.ts

```diff
@@ -331,6 +333,11 @@ declare module Elasticsearch {
         refresh?: boolean;
     }
 
+    export interface DeleteByQueryParams extends GenericParams {
+        index: string;
```

Comment:
Hi,

it is not possible to use the ```waitForActiveShards``` option with deleteByQuery either. Maybe there is a documentation mismatch.

I disagree with your opinion regarding the coupling. DeleteByQuery is meant to be used the same way as search.

Cheers,
Finn